sale_order_report_hide_discounts_by_partner/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2025 Ángel García de la Chica <angel.garcia@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Order Report Hide Discounts by Partner",
+    "summary": "Sale Order Report Hide Discounts by Partner",
+    "version": "16.0.1.0.0",
+    "category": "Sale",
+    "website": "https://github.com/sygel-technology/sy-sale-reporting",
+    "author": "Sygel, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "account",
+        "base",
+        "sale",
```

Comment:
Yo just need "sale" as a dependency

Reply:
I thought that in a custom module when you have a reference to another module you have to put it as a dependency (although it may be redundant with respect to other dependencies), but maybe I'm wrong. In this case I have put it:
account -> for the tax groups account.groups....
base -> for the view base.view_partner_form

Reply:
There's not need really, and it's not wrong to include them. It's just to keep the list as simple as possible.